qa/os/src/test/java/org/elasticsearch/packaging/test/PackagingTestCase.java

```diff
@@ -701,4 +701,17 @@ public static void verifySecurityNotAutoConfigured(Installation es) throws Excep
         }
     }
 
+    public void updateConfig(Path confPath, String key, String replacement) throws IOException {
```

Comment:
There are some methods in ServerUtils for adding/removing settings. Perhaps this should go there? Or those existing methods could be used instead.

Reply:
Thanks for the suggestion! I'll take a look to see if I can reuse something in there.